numba/cuda/tests/cudapy/test_compiler.py

```diff
@@ -30,8 +30,8 @@ def test_device_function(self):
         def add(x, y):
             return x + y
 
-        args = (float32, float32)
-        ptx, resty = compile_ptx(add, args, device=True)
+        sig = (float32, float32)
+        ptx, resty = compile_ptx(add, sig, device=True)
```

Comment:
Since you're passing a tuple of args and not a signature object, I think this change makes the code less succinct - I'd suggest going back to its original form:

```suggestion
        args = (float32, float32)
        ptx, resty = compile_ptx(add, args, device=True)
```